redfish-core/lib/log_services.hpp

```diff
@@ -1968,76 +2390,72 @@ inline void requestRoutesDBusEventLogEntryDownload(App& app)
                 std::string entryID = param;
                 dbus::utility::escapePathForDbus(entryID);
 
-                crow::connections::systemBus->async_method_call(
-                    [asyncResp,
-                     entryID](const boost::system::error_code ec,
-                              const sdbusplus::message::unix_fd& unixfd) {
-                        if (ec.value() == EBADR)
-                        {
-                            messages::resourceNotFound(
-                                asyncResp->res, "EventLogAttachment", entryID);
-                            return;
-                        }
-                        if (ec)
-                        {
-                            BMCWEB_LOG_DEBUG << "DBUS response error " << ec;
-                            messages::internalError(asyncResp->res);
-                            return;
-                        }
+                auto getHiddenProperty = [asyncResp,
+                                          entryID](bool hiddenPropVal) {
+                    if (hiddenPropVal)
+                    {
+                        messages::resourceNotFound(asyncResp->res, "LogEntry",
+                                                   entryID);
+                        return;
+                    }
+                    getEventLogEntryAttachment(asyncResp, entryID);
+                };
+                getHiddenPropertyValue(asyncResp, entryID,
+                                       std::move(getHiddenProperty));
+            });
+}
 
-                        int fd = -1;
-                        fd = dup(unixfd);
-                        if (fd == -1)
-                        {
-                            messages::internalError(asyncResp->res);
-                            return;
-                        }
+inline void requestRoutesDBusCELogEntryDownload(App& app)
+{
+    BMCWEB_ROUTE(app, "/redfish/v1/Systems/system/LogServices/CELog/Entries/"
+                      "<str>/attachment")
+        .privileges(redfish::privileges::getCELogEntry)
```

Comment:
Just do         .privileges({{"ConfigureManager"}}) 
with a comment saying we are overrating the normal permissions for a LogEntry

Reply:
Note: I don't think it is a requirement we overwrite them (make it admin only) but it is fine if we do

Reply:
Done